content/docs/rendering-elements.md

```diff
@@ -8,68 +8,68 @@ prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
-
-An element describes what you want to see on the screen:
+एलिमेंट्स एक रिएक्ट ऐप्स के सबसे छोटे बिल्डिंग ब्लॉक हैं।
 
+एक एलिमेंट वर्णन करता है कि आप स्क्रीन पर क्या देखना चाहते हैं:      
+
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+ब्राउज़र DOM एलेमेंट के विपरीत, React एलेमेंट्स एक सादा ऑब्जेक्ट है और इन्हें बनाना आसान है। React DOM React एलेमेंट्स के अनुसार DOM को अद्यतन बनाए रखता है।
```

Comment:
`एलेमेंट` => `एलेमेंट्स`

Reply:
`एलेमेंट्स` => `एलिमेंट्स`

actually this sounds more relevant, can you change this everywhere

Reply:
You missed this one